site/public/js/user-profile.js

```diff
@@ -1,10 +1,10 @@
 /* exported updateUserPronouns, showUpdatePrefNameForm, showUpdatePronounsForm,
 showUpdatePasswordForm, showUpdateProfilePhotoForm, showUpdateSecondaryEmailForm,
 updateUserPreferredNames, updateUserProfilePhoto, updateUserSecondaryEmail,
-changeSecondaryEmail
+changeSecondaryEmail, clearPronounsBox
  */
 
-var pronounsLastVal = null;
+let pronounsLastVal = null;
```

Comment:
Is there an easy way to avoid having a global variable here?

Reply:
We discussed